src/main/java/uk/gov/companieshouse/api/accounts/service/impl/DebtorsService.java

```diff
@@ -38,24 +39,35 @@
     private DebtorsTransformer transformer;
     private SmallFullService smallFullService;
     private KeyIdGenerator keyIdGenerator;
+    private DebtorsValidator debtorsValidator;
 
     @Autowired
     public DebtorsService(DebtorsRepository repository,
                           DebtorsTransformer transformer,
                           SmallFullService smallFullService,
-                          KeyIdGenerator keyIdGenerator) {
+                          KeyIdGenerator keyIdGenerator, DebtorsValidator debtorsValidator) {
 
         this.repository = repository;
         this.transformer = transformer;
         this.smallFullService = smallFullService;
         this.keyIdGenerator = keyIdGenerator;
+        this.debtorsValidator = debtorsValidator;
     }
 
     @Override
     public ResponseObject<Debtors> create(Debtors rest, Transaction transaction,
                                           String companyAccountsId, HttpServletRequest request)
         throws DataException {
 
+        Errors errors;
```

Comment:
Could the declaration for `errors` and the call to populate `errors` be on the same line?